src/wwElement.vue

```diff
@@ -225,7 +225,7 @@ export default {
     },
     mounted() {
         this.handleObserver();
-        this.handleOpening(this.content.isOpen);
+        this.handleOpening(this.wwEditorState.sidepanelContent.openInEditor);
```

Comment:
Risque pas d'y avoir une erreur ici dans le site publié ? Comme wwEditorState existera pas t'auras sidepanelContent of undefined

Reply:
Yes bien vu !